x-pack/plugins/security_solution/server/lib/detection_engine/rules/types.ts

```diff
@@ -165,9 +172,11 @@ export const isRuleStatusFindTypes = (
 export interface CreateRulesOptions {
   alertsClient: AlertsClient;
   anomalyThreshold: AnomalyThresholdOrUndefined;
+  author: AuthorOrUndefined;
+  buildingBlockType: BuildingBlockTypeOrUndefined;
   description: Description;
   enabled: Enabled;
-  falsePositives: FalsePositives;
+  falsePositives: FalsePositives; // TODO: Why isn't this FalsePositivesOrUndefined?
```

Comment:
Remove this TODO, it is not valid:

```ts
false_positives: DefaultStringArray, // defaults to empty string array if not set during decode
```

That's in the create_rules_schema. It's an empty array when you are creating a rule, no? Double check me but this TODO is not valid and will mislead someone into typing this wrong.

Reply:
That's what I wanted to verify -- thanks! Wasn't sure why it wasn't `OrUndefined` since it's not a required field, but looks like the UI is just sending up an empty array if not set.